### PR TITLE
chore: Update amazon-bedrock tile for AmazonBedrockChatGenerator

### PR DESCRIPTION
**Pull request by vblagoje**
Add doc usage example for AmazonBedrockChatGenerator

### CODE REVIEW

File: integrations/amazon-bedrock.md [hunk 1]
```diff
@@ -64,3 +65,32 @@ Output:
...
+
+To use this integration for chat models, initialize a `AmazonBedrockChatGenerator` with the model name and aws credentials:
+
+Currently, the following models are supported: 
+- Anthropic's Claude
+- Meta's Llama 2
+
+
+```python
+from haystack_integrations.components.generators.amazon_bedrock import AmazonBedrockChatGenerator
```
Comment (annthurium) [inline integrations/amazon-bedrock.md:79]: I'm getting an error here --
```

ImportError: cannot import name 'AmazonBedrockChatGenerator' from 'haystack_integrations.components.generators.amazon_bedrock' (/usr/local/lib/python3.10/dist-packages/haystack_integrations/components/generators/amazon_bedrock/__init__.py)
```

probably doing something obvious wrong - I ran `pip install amazon-bedrock-haystack` and `haystack-ai` but is there another depencency that's missing or something?

Comment (vblagoje) [reply integrations/amazon-bedrock.md:79]: @annthurium haven't published pypi release yet. Waiting for docs to be finished by Daria etc
But please try from main directly and TIA 🙏

Comment (annthurium) [reply integrations/amazon-bedrock.md:79]: @vblagoje is there a way to install a haystack integration from main other than git cloning the whole repo and trying to run it locally? Sorry, I haven't had to do this yet and it's not documented anywhere I could find

Comment (vblagoje) [reply integrations/amazon-bedrock.md:79]: I'm not 100% - I always run it directly from a clone of haystack-core-integrations project where I can change branches. Perhaps that is the best for dev setup...

**Review by annthurium — APPROVED** [review]

**Pull request merged**